config/core/test-configs.yaml

```diff
@@ -2147,6 +2154,7 @@ test_configs:
       - ltp-pty
       - ltp-timers
       - sleep
+      - smc
```

Comment:
This is not working, I think it's because the ramdisk + kernel image size goes over the limit for the TFTP download on this platform:
https://lava.collabora.co.uk/scheduler/job/4252350

@aliceinwire Can you change the rootfs to buildroot_ramdisk, if you don't need anything specific in Debian?  The buildroot images are smaller.

Reply:
I see you already sended a pull request for it ?

Reply:
Yes as a draft to try things out on staging.  Actually I think there's an issue with the `hana` device type, it should be fixed to be able to download bigger ramdisks.  In principle the limit is 128MB for the kernel image + ramdisk, but it seems like that's not the case in the current firmware.  I think we can stick to Debian and just not run the tests on `hana` for now, but let's see whether it works with buildroot first.

Reply:
ok thanks. I will update this pull request

Reply:
This was tested today on staging with a full linux-next build, and the `hana` failed even with buildroot:
https://staging.kernelci.org/test/plan/id/6103b8fc73ff9edaffd74f47/

So please drop this device type for now.  Someone at Collabora will look into the issue, we know it's probably due to a size limitation in the firmware with TFTP downloads.